app/src/main/res/layout/on_boarding_activity.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:gravity="center"
+  android:orientation="vertical">
+
+  <Button
+    android:id="@+id/onBoard"
```

Comment:
In id's we are following all small caps, update this to `on_board`

Reply:
Done.